sw/device/tests/crypto/cryptotest/firmware/kmac_sca.c

```diff
@@ -327,18 +344,38 @@ static kmac_sca_error_t kmac_msg_write(const void *msg, size_t msg_len,
 /**
  * Starts actual processing of a previously provided message.
  *
- * This function issues a START command directly followed by a PROCESS command.
+ * This function issues a PROCESS command.
  */
-static void kmac_msg_proc(void) {
+static void kmac_process_cmd(void) {
+  // Issue PROCESS command.
+  uint32_t cmd_reg =
+      bitfield_field32_write(0, KMAC_CMD_CMD_FIELD, KMAC_CMD_CMD_VALUE_PROCESS);
+  mmio_region_write32(kmac.base_addr, KMAC_CMD_REG_OFFSET, cmd_reg);
+}
+
+/**
+ * Starts actual processing of a previously provided message.
+ *
+ * This function issues a PROCESS command.
+ */
```

Comment:
Nit: this comment seems wrong. The start command just prepares KMAC to receive the message via message FIFO.

Reply:
Thanks, I've changed the comment.